containers/daap-get-glue-metadata/src/var/task/get_glue_metadata.py

```diff
@@ -0,0 +1,12 @@
+import boto3
+import json
+
+
```

Comment:
```suggestion
import json

import boto3
```

From https://github.com/ministryofjustice/data-platform/actions/runs/5585558508/jobs/10208449148

Reference: https://pycqa.github.io/isort/